Add benchmarks from rv8-bench and nbench

qwe661234 opened the pull request:
These benchmarks are compiled from `rv8-bench` and `nbench` using the compiler `riscv32-unknown-elf-gcc`, which incorporates RV32IMC extensions, and the optimization level is set to O2.

jserv reviewed (changes requested):
Update `README.md` for the corresponding information.

qwe661234 committed:
Add benchmarks for evaluating performance

These benchmarks are compiled from rv8-bench and nbench using the compiler
riscv32-unknown-elf-gcc, which incorporates RV32IMC extensions, and the
optimization level is set to O2.
